test cases/failing/89 link_with custom target/lib_generator.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Mimic a binary that generates a static library
+
+import sys, subprocess
+
+if __name__ == '__main__':
+    if len(sys.argv) != 4:
+        print(sys.argv[0], 'compiler input_file output_file')
+        sys.exit(1)
+    compiler = sys.argv[1]
+    ifile = sys.argv[2]
+    ofile = sys.argv[3]
+    if compiler.endswith('cl'):
+        cmd = [compiler, '/nologo', '/MDd', '/Fo' + ofile, '/c', ifile]
+    else:
+        cmd = [compiler, '-c', ifile, '-o', ofile]
```

Comment:
This does not create an `.a` output file, it only creates an `.o` object file. You need to call the static linker too.

Reply:
Indeed.